src/GraphQl/Action/EntrypointAction.php

```diff
@@ -54,11 +54,11 @@ public function __invoke(Request $request): Response
         list($query, $operation, $variables) = $this->parseRequest($request);
 
         if (null === $query) {
-            return new JsonResponse(new ExecutionResult(null, [new Error('GraphQL query is not valid')]), Response::HTTP_BAD_REQUEST);
```

Comment:
Aren't this two ones legit? This time we're sure it's a bad request.

Reply:
When there is no "mixed" result, we can safely return a 400. It's the case here.

Reply:
I don't agree. @bwegrzyn thought he could use status code whereas it was only for some cases. This is very misleading and I think we should not use them at all.

Reply:
From an HTTP POV, Apollo choice makes sens:

- the HTTP request is entirely invalid => 400
- the HTTP request is partially valid, but for some instructions an error is generated => 200